### ISSUE TITLE
Show more events during start of process

### ISSUE THREAD

**Issue by maralorn** [maralorn/nix-output-monitor#73]
It might be interesting to show users how long the process is evaluating and how long it is querying substituters.

**Comment by takeda** [comment]
Yes, please. Not sure why but on my machine for some reason this step doesn't take trivial amount of time, and NOM doesn't state what is happening.
